data-explorer/kusto/query/toc.yml

```diff
@@ -507,6 +507,8 @@ items:
     href: hllmergefunction.md
   - name: hourofday()
     href: hourofdayfunction.md
+  - name: iff()
```

Comment:
I'm not sure I understand the addition- this entry already exists on lines 512 and 513